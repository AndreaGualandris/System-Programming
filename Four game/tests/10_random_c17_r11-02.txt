10
12
12
10
16
3
10
11
11
4
2
14
9
6
10
0
6
3
0
2
4
12
14
13
8
1
10
5
14
4
5
5
7
13
11
12
5
9
10
16
14
11
1
9
2
4
12
0
8
4
1
1
4
11
11
0
5
14
10
14
3
12
6
11
0
14
10
15
16
8
6
1
11
0
9
4
13
2
11
16
12
12
2
8
6
9
11
3
8
9
11
6
16
12
4
3
14
16
8
8
3
15
7
0
16
9
6
13
16
11
7
15
5
5
3
7
5
13
9
6
16
13
2
7
1
6
9
2
0
11
15
10
1
6
1
13
10
0
3
2
9
8
11
2
9
14
2
1
11
5
3
14
11
15
4
12
3
0
5
6
0
7
9
6
2
0
6
6
14
6
3
4
16
5
11
10
10
1
14
11
12
0
6
15
4
4
5
