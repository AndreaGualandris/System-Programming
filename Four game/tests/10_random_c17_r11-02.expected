-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
          o      
-----------------
01234567890123456
Player 'x' to move:
            x
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
                 
          o x    
-----------------
01234567890123456
Player 'o' to move:
            o
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
            o    
          o x    
-----------------
01234567890123456
Player 'x' to move:
          x
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
          x o    
          o x    
-----------------
01234567890123456
Player 'o' to move:
                o
                v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
          x o    
          o x   o
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
                 
          x o    
   x      o x   o
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          x o    
   x      o x   o
-----------------
01234567890123456
Player 'x' to move:
           x
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          x o    
   x      oxx   o
-----------------
01234567890123456
Player 'o' to move:
           o
           v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
   x      oxx   o
-----------------
01234567890123456
Player 'x' to move:
    x
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
   xx     oxx   o
-----------------
01234567890123456
Player 'o' to move:
  o
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
  oxx     oxx   o
-----------------
01234567890123456
Player 'x' to move:
              x
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
  oxx     oxx x o
-----------------
01234567890123456
Player 'o' to move:
         o
         v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
  oxx    ooxx x o
-----------------
01234567890123456
Player 'x' to move:
      x
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
                 
          o      
          xoo    
  oxx x  ooxx x o
-----------------
01234567890123456
Player 'o' to move:
          o
          v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
          xoo    
  oxx x  ooxx x o
-----------------
01234567890123456
Player 'x' to move:
x
v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
          xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'o' to move:
      o
      v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
      o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'x' to move:
   x
   v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
   x  o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'o' to move:
o
v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
o  x  o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'x' to move:
  x
  v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
o xx  o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'o' to move:
    o
    v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o      
o xxo o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'x' to move:
            x
            v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o x    
o xxo o   xoo    
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'o' to move:
              o
              v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o x    
o xxo o   xoo o  
x oxx x  ooxx x o
-----------------
01234567890123456
Player 'x' to move:
             x
             v
-----------------
                 
                 
                 
                 
                 
                 
                 
          o      
          o x    
o xxo o   xoo o  
x oxx x  ooxxxx o
-----------------
01234567890123456
Player 'x' wins!
